dataeng/jobs/analytics/SnowflakeMicrobachelorsITK.groovy

```diff
@@ -23,6 +23,7 @@ class SnowflakeMicrobachelorsITK {
             parameters {
                 stringParam('ANALYTICS_TOOLS_URL', allVars.get('ANALYTICS_TOOLS_URL'), 'URL for the analytics tools repo.')
                 stringParam('ANALYTICS_TOOLS_BRANCH', allVars.get('ANALYTICS_TOOLS_BRANCH'), 'Branch of analytics tools repo to use.')
+                stringParam('NOTIFY', allVars.get('NOTIFY','$PAGER_NOTIFY_APERTURE'), 'Space separated list of emails to send notifications to.')
```

Comment:
Sorry to be a little confused here.   Is there a reason to specify the Aperture pager email address as an environment variable on the Jenkins server, rather than as a configuration parameter in the job.   This has an allVars.get('NOTIFY'), so it should get the value from the job settings in analytics-secure.   But that should be enough, in my view, rather than wiring a particular transient team name up through Jenkins and the configuration repo.   (And just clear the default here, I guess.)   And if you include publishers, would you need this line at all anymore?

Reply:
I understand your motivation entirely.  I'm just concerned whether following the example of PAGER_NOTIFY is a good way to go for DE and non-DE groups alike.  And I'm checking with DE on this.   But so far we have Enterprise jobs having their NOTIFY value set to "enterprise-integrations@edx.org", and program enrollment report failures being sent to "program-analytics@edx.opsgenie.net", using NOTIFY values in analytics-secure.  So there is already precedence.   

If there is a concern about cut-and-paste, then we can define a value similar to the value of ANALYTICS_EXPORTER_NOTIFY_LIST, which is also defined in a common.yaml file in analytics-secure and shared across several exporter-related jobs.    The downside of this approach is that it has to be fetched as allVars.get('ANALYTICS_EXPORTER_NOTIFY_LIST'), so it has to be done in the groovy code.   This is fine for exporter, where the notifications are the same independent of environment, but it doesn't work for per-environment configuration.    It means having to use your approach below of moving per-environment configuration up into the job definition.  

Either way, I'd prefer not to use the Jenkins environment variable approach.   And hope to confirm that with DE tomorrow morning.

Reply:
Yes, the problem is compounded by the fact that you don't have visibility into the analytics-secure repo.